### PR TITLE
Ge bug hunt

### LINKED ISSUE #56: Option to choose a new prompt if you don't like the current one

### LINKED ISSUE #58: Fix bookshelf hovers

### LINKED ISSUE #60: Error message for incorrect password on login

### PR DESCRIPTION
**Pull request by GreysonElkins**
#### What's this PR do?
* Add error message for incorrect password
* Show prompts before start writing 
* Style new hover
* Fix last words
* Add home button / refresh current genre prompt button
#### Where should the reviewer start?
Whereever you want!
#### How should this be manually tested?
I apologize that I've fallen to the way-side with tests over the last few days :(
#### Any background context you want to provide?
* Prompts are now grabbed before submit
* Also, sorry @carlymclift, I looked at the docs and after what you said about html in tool-tip I thought it'd be quicker to just scrape the package and write it differently.
  * I tried to uninstall the package and it broke the site. Idk what to do about that. 
#### What are the relevant tickets?
close #60, close #58, close #56
#### Questions:
* unrelated but **FOR LATER**: How can we check that all our packages are saved and installing correctly with `npm install`? I know this was a problem on my last project when Khalid tried to run things locally.

### CODE REVIEW

**Review by carlymclift — APPROVED** [review]
Excellent

File: src/components/App/App.js [hunk 1]
```diff
@@ -98,8 +100,39 @@ class App extends Component {
...
+            <br />
+          </span>
+          <span>
+            LAST UPDATED: {this.state.hover.lastUpdate}
+          </span>
+        </p>
+      </div>
+    )
+  }
+
```
Comment (carlymclift) [inline src/components/App/App.js:132]: great work fixing this hover functionality

**Review by nickhartdev — APPROVED** [review]
Thanks for getting these issues knocked out!

File: src/components/App/App.js [hunk 2]
```diff
@@ -38,7 +40,7 @@ class App extends Component {
               }
               if (story.contributions[0] !== null) {
                 const lastEntry = story.contributions[story.contributions.length - 1];
-                story.lastWords = `. . . ${lastEntry.substring(15)}`;
+                story.lastWords = `. . . ${lastEntry.slice(-150)}`;
               }
```
Comment (Abdeboskey) [inline src/components/App/App.js:44]: sweet, i think that's a good slice size

File: src/components/App/App.js (hunk 1 above)
Comment (Abdeboskey) [reply src/components/App/App.js:132]: Very cool fo sho

File: src/components/Header/Header.js [hunk 3]
```diff
@@ -12,6 +12,9 @@ const Header = () => {
         </h1>
       </Link>
       <nav className='navbar'>
+        <NavLink to='/' className='nav' activeClassName='selected'>
+          / Home / 
+        </NavLink>
         <NavLink to='/story-setup' className='nav' activeClassName='selected'>
          /  New Story /
         </NavLink>
```
Comment (Abdeboskey) [inline src/components/Header/Header.js:20]: Perfect, thanks!

File: src/components/Instructions/Instructions.js [hunk 4]
```diff
@@ -46,11 +46,12 @@ const Instructions = ( { displayHistory, showHistory, stories, onClick, authorUp
...
-            stories={stories}
-            onClick={onClick}
-            authorUpdater={authorUpdater}
-          />
+        <Bookshelf
+          stories={stories}
+          onClick={onClick}
+          authorUpdater={authorUpdater}
+          toggleHover={toggleHover}
+        />
```
Comment (Abdeboskey) [inline src/components/Instructions/Instructions.js:54]: that's a nice lil method!

File: src/components/StorySetup/StorySetup.js [hunk 5]
```diff
@@ -67,18 +76,24 @@ class StorySetup extends Component {
...
+          <br />
           Have Fun!
         </p>
         <button>Start Story</button>
+        {this.props.randomPrompt 
+          && <span class="prompt">
+            PROMPT: {this.props.randomPrompt.prompt}
+            <br />
+          </span>
+        }
```
Comment (Abdeboskey) [inline src/components/StorySetup/StorySetup.js:94]: Is the prompt still displayed on story edit view?

**Review by Abdeboskey — APPROVED** [review]
Nice fixes/features here bubs!

File: src/components/StorySetup/StorySetup.js (hunk 5 above)
Comment (GreysonElkins) [reply src/components/StorySetup/StorySetup.js:94]: Yup, should be!

**Pull request merged**